python/neuroglancer/local_volume.py

```diff
@@ -193,7 +193,9 @@ def get_encoded_subvolume(self, data_format, start, end, scale_key):
             or np.prod(downsample_factor) > self.max_downsampling
         ):
             raise ValueError("Invalid downsampling factor.")
-        downsampled_shape = np.cast[np.int64](np.ceil(self.shape / downsample_factor))
+        downsampled_shape = np.asarray(
+            np.ceil(self.shape / downsample_factor, dtype=np.int64)
```

Comment:
I'm running into a snag here. 

`TypeError: No loop matching the specified signature and casting was found for ufunc ceil`

I moved the type to the asarray and it seemed to have worked. 
`        downsampled_shape = np.asarray(
            np.ceil(self.shape / downsample_factor), dtype=np.int64
        )` 
I believe it has to do with the

Reply:
thanks fixed!